tools/crm_mon_print.c

```diff
@@ -163,7 +163,7 @@ print_resources_closing(pcmk__output_t *out, unsigned int mon_ops)
 static int
 print_resources(pcmk__output_t *out, pe_working_set_t *data_set,
                 unsigned int print_opts, unsigned int mon_ops, gboolean brief_output,
-                gboolean print_summary, gboolean print_spacer)
+                gboolean print_summary, gchar **allowed_nodes, gboolean print_spacer)
```

Comment:
Let's name this something different, we use "allowed_nodes" in the scheduler to mean nodes that the resource is allowed to run on

Reply:
Renamed to only_show, which is what I was using before.